### DISCUSSION TITLE
general question

### DISCUSSION THREAD

**Discussion by rishigarg256** [smartcontractkit/full-blockchain-solidity-course-py#1165] [Q&A]
whenever i try to type a function like wait_fot_transaction_receipt, my vs code does not give me that suggestion and i have to type it out fully. what should i do so that i also get the suggestions for the w3 functions?

**Comment by cromewar** [comment]
There is not a native support for that type of completions right now, you can use something like:

- TabNine: IA autocompletion extension for Vscode
- Visual Studio Intellicode extension by Microsoft.
